packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -686,6 +686,13 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     );
   }
 
+  // Unsupported
+  // TODO(MxSoul): see https://github.com/flutter/flutter/issues/100748
```

Comment:
What is the TODO here? To implement it? Let's mention that in the TODO comment.

Reply:
(and why isn't the PR just implementing this to avoid adding more tech debt)?

Reply:
Because these are two different scenarios, and they're not coupled, we can probably solve them one by one